src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
-body {
+$main-color: #060b35;
+$secondary-color: #616070;
+$white-color: #fff;
+$lightblue-color: #00acdc;
+
+* {
   margin: 0;
+  padding: 0;
```

Comment:
for the * { } we can set only box-sizing. So you need to move padding: 0 and margin: 0 to the tag body